src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -238,6 +216,95 @@ private void initializeAppsLocation(CommonLoggerI log, File serverXML, File conf
         }
     }
 
+
+    /**
+     * jvm.options file read order
+     *   1. {wlp.user.dir}/
+     *   2. {server.config.dir}/configDropins/defaults/
+     *   3. {server.config.dir}/
+     *   4. {server.config.dir}/configDropins/overrides/
+     * @throws FileNotFoundException
+     * @throws Exception
+     */
+    public void processJvmOptions() throws FileNotFoundException, Exception {
+        final String jvmOptionsString = "jvm.options";
+        parsePropertiesFromFile(new File(libertyDirectoryPropertyToFile.get(ServerFeatureUtil.WLP_USER_DIR), jvmOptionsString));
```

Comment:
The Liberty doc says the first location should be `{wlp.user.dir}/shared`. So I think you need to tack on `shared/` before `jvm.options`.